app/modules/individuals/tasks.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+import app.extensions.logging as AuditLog
+import logging
+from app.extensions.celery import celery
+
+log = logging.getLogger(__name__)
+
+
+@celery.task(
+    bind=True,
+    # autoretry_for=(requests.exceptions.RequestException,),
+    # default_retry_delay=3,
+    # max_retries=10,
+)
+def execute_merge_request(self, target_individual_guid, from_individual_ids, parameters):
```

Comment:
The general pattern in other tasks is to do the minimum in the the task function as possible and have the functionality implemented in a model method. 
I'd suggest you should follow this pattern here too.

Reply:
gotcha.  [done](https://github.com/WildMeOrg/houston/pull/365/commits/7da861d91c8db39c6e936a955a7baad927ac27e3).